src/isl/isl_type_reference.rs

```diff
@@ -13,17 +13,32 @@ use ion_rs::IonType;
 #[derive(Debug, Clone, PartialEq)]
 pub enum IslTypeRef {
     /// Represents a reference to one of ISL's built-in core types:
-    CoreIslType(IonType),
+    CoreIsl(IonType),
```

Comment:
I have a mild preference for naming this variant `Core`; the spec already refers to them as "[Core Types](https://amzn.github.io/ion-schema/docs/spec.html#core-types)".

Reply:
Yes, I think that should be fine as far as it is similar to what the spec says.